kubernetes/gke-utility/argocd/extras.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kubernetes-repo
+  namespace: argocd
+  labels:
+    argocd.argoproj.io/secret-type: repository
+stringData:
+  url: https://github.com/kubernetes
+  name: kubernetes
+  type: git
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: apps
+spec:
+  destination:
+    namespace: argocd
+    server: https://kubernetes.default.svc
+  project: default
+  source:
+    path: kubernetes/apps
+    repoURL: https://github.com/borg-land/k8s.io
+    targetRevision: utility-dev
+  syncPolicy:
+    automated:
+      prune: false
```

Comment:
I see `prune: false` used in a couple of places, but we have to be careful with it because it can leak resources and folks might not expect that. I think we should have a note in the README file describing that some resources are not pruned and that it must be done manually.

Reply:
ArgoCD will be the one service where pruning is disabled. All other deployments will be automated with pruning enabled.